From: Beniamino Galvani <bgalvani@redhat.com>
Date: Wed, 31 Oct 2018 11:29:26 +0100
Subject: cli/tests: fix output on failure

(cherry picked from commit 5665f67bae25ee9f96e057c2a993b8cf69cbc127)
(cherry picked from commit d1afd1c4206d3720de7776ed93ae61c48a7cc71d)
---
 clients/tests/test-client.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clients/tests/test-client.py b/clients/tests/test-client.py
index e8f1f7f..d92d499 100755
--- a/clients/tests/test-client.py
+++ b/clients/tests/test-client.py
@@ -744,8 +744,8 @@ class TestNmcli(NmTestBase):
                     skip_test_for_l10n_diff.append(n['test_name'])
                     continue
                 print("\n\n\nThe file '%s' does not have the expected content:" % (filename))
-                print("ACTUAL OUTPUT:\n[[%s]]\n" % (results_expect[i]))
-                print("EXPECT OUTPUT:\n[[%s]]\n" % (n['content']))
+                print("ACTUAL OUTPUT:\n[[%s]]\n" % (n['content']))
+                print("EXPECT OUTPUT:\n[[%s]]\n" % (results_expect[i]))
                 print("Let the test write the file by rerunning with NM_TEST_REGENERATE=1")
                 print("See howto in %s for details.\n" % (PathConfiguration.canonical_script_filename()))
                 self.fail("Unexpected output of command, expected %s. Rerun test with NM_TEST_REGENERATE=1 to regenerate files" % (filename))
